core/src/data/row.rs

```diff
@@ -63,6 +66,14 @@ impl Row {
             return Err(RowError::TooManyValues.into());
         }
 
+        if let Some(wrong_column_name) = columns.iter().find(|column_name| {
+            !column_defs
+                .iter()
+                .any(|column_def| &&column_def.name == column_name)
+        }) {
+            return Err(RowError::WrongColumnName(wrong_column_name.to_owned()).into());
```

Comment:
To resovle case like
```sql
gluesql> CREATE TABLE Items (id INT NULL, name TEXT NULL);
Table created

gluesql> INSERT INTO Items (status) VALUES (1);
1 row inserted
```
It's so confused whether validate here or not

Reply:
I just had missed more test case. It's reachable! even if i validate at the top!